main.go

```diff
@@ -198,10 +198,6 @@ func setupChecks(mgr ctrl.Manager) {
 }
 
 func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
```

Comment:
Let's move this back inline, same goes for the setupWebhooks

Reply:
This applies to all the other managers as well

Reply:
Leaving this as it is for better code orgazination